src/components/OutputData.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import { t } from 'ttag';
+
+/**
+ * It renders a card containing a input field for the data cotent as string,
+ * and an action to remove the card.
+ *
+ * @param {Object} props
+ * @property {MutableRefObject} dataInputRef Reference for input field value
+ * @property {number} index Data output position in the list
+ * @property {(index: number) => void} remove Callback to remove the data output from the list
+ */
+export const OutputData = ({ dataInputRef, index, remove }) => (
```

Comment:
Maybe we should name the same way we name the component to add a token output.
Something like `SendDataOne` or `SendDataOutputOne` what do you think?

Reply:
Renamed to `SendDataOutputOne`...